tests/kafkatest/services/performance/producer_performance.py

```diff
@@ -43,21 +50,33 @@ def __init__(self, context, num_nodes, kafka, security_protocol, topic, num_reco
         self.intermediate_stats = intermediate_stats
         self.client_id = client_id
 
+    def pids(self, node):
+        try:
+            cmd = "ps ax | grep -i ProducerPerformance | grep java | grep -v grep | awk '{print $1}'"
+            pid_arr = [pid for pid in node.account.ssh_capture(cmd, allow_fail=True, callback=int)]
+            return pid_arr
+        except (subprocess.CalledProcessError, ValueError) as e:
+            return []
+
+    def alive(self, node):
+        return len(self.pids(node)) > 0
+
     def _worker(self, idx, node):
         args = self.args.copy()
-        args.update({'bootstrap_servers': self.kafka.bootstrap_servers(), 'jmx_port': self.jmx_port, 'client_id': self.client_id})
-        cmd = "JMX_PORT=%(jmx_port)d /opt/kafka/bin/kafka-run-class.sh org.apache.kafka.clients.tools.ProducerPerformance " \
-              "%(topic)s %(num_records)d %(record_size)d %(throughput)d bootstrap.servers=%(bootstrap_servers)s client.id=%(client_id)s" % args
+        args.update({'bootstrap_servers': self.kafka.bootstrap_servers(), 'client_id': self.client_id})
+
+        cmd = "export JMX_PORT=%s; " % str(self.jmx_port)
```

Comment:
Do these all actually need to be exported separately instead of just being specified inline as was done previously? Just curious since it seems to be a consistent change you're making across a bunch of services.